app/experimenter/static/js/experiment-date-filter.js

```diff
@@ -0,0 +1,27 @@
+let dateRangeSelector = document.getElementById('id_experiment_date_field');
+let rangeElementInput0 = document.getElementById("id_date_range_0");
+let rangeElementInput1 = document.getElementById("id_date_range_1");
+console.log(dateRangeSelector);
+
+function enableBothInputs () {
+  rangeElementInput0.removeAttribute("disabled");
+  rangeElementInput1.removeAttribute("disabled");
+}
+
+function setUpDateRange () {
+  if (dateRangeSelector.value == 'starts_between') {
+    enableBothInputs();
+  } else if (dateRangeSelector.value == 'pauses_between') {
+    enableBothInputs();
+  } else if (dateRangeSelector.value == 'ends_between') {
+    enableBothInputs();
+  } else {
```

Comment:
It might be easier to check for the default "empty" value and enable the fields otherwise.

Reply:
yep you're right.